src/pages/_layouts/default.html

```diff
@@ -19,7 +19,7 @@
 		{% include layout/page-header.html %}
 
 		<!-- Page body -->
-		<div class="page-body">
+		<main class="page-body{% if page.page-body-class %} {{ page.page-body-class }}{% endif %}">
```

Comment:
May I ask: is there a strong reason for that change? It seems like a BC break, especially regarding custom CSS rules.

Reply:
it's temporary

Reply:
I never came across the `main` element, but adapting to the standards is not a bad idea. but: this is 100% sure a BC break. Even in my own projects I found a `document.querySelector('div.page-body')` and I am only one of now 32k stargazers^^ 

So: revert `main` vs `div` or keep it?

Reply:
I reverted it, it was only temporary